src/main/java/service/SourceHandler.java

```diff
@@ -0,0 +1,7 @@
+package service;
+
+public interface SourceHandler {
+    boolean readInputData(String path);
+
+    boolean writOutputData(String data, String path);
```

Comment:
You lose the letters inside the words, let's be more careful
```suggestion
    boolean write(String data, String path);
```